data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1613 @@
+			label curious
+			`	She gestures to the ship next to her. "You have arrived in our space at an opportune time, <first>. While most of our developments over the years have been tweaking existing ships, tweaks can only go so far. Even before we established our first shipyard, we have been constantly innovating and trying out new ship designs. Every new development is put on the conceptual model to see if it can help us build a better ship." She grimaces slightly. "Well, that is a bit hyperbolic. Not every development applies to spacecraft, just the ones of most interest to me." She grins at that. "In any case, several of our development projects are nearing the point of production, and this prototype should be operational within a few days."`
+			`	She pats a piece of exposed hull, and you think you can almost feel some kind of vibration or hum in response. Taely takes on a distant expression, as if lost in thought. After a long enough pause to feel awkward, she blinks a few times and focuses on you again. "Sorry about that. A new ship waking up and all that." She smiles, thoughtfully. "Before I get absorbed in my work again, would you be able to make a delivery for me? It is a simple one, but important. I need someone to deliver a crate of data files to <destination>. As my development project has reached its end, it is time to send a hard copy of the data to one of our archives for safe keeping. Could you deliver it for me?"`
+			choice
+				`	"Yes, I can"`
+					goto affirmative
+				`	"I am too busy for that."`
+			`	Taely looks disappointed, but shrugs. "No problem, I will get someone else to deliver it."`
+				decline
+			label affirmative
+			`	Taely looks please. "Excellent. Just delivers this to the archive on <destination>." She hands you a very robust looking datacrystal. "Looks a bit different from other data crystals, I am sure." She pauses. "These are archival crystals, and as such they have more robust data structures with many more layers of redundancy and error correction. Likewise, they have many additional layers of encryption and security to ensure that it does not fall into enemy hands. In other words, perfect for long-term storage." She pauses and smiles. "Once you have delivered this, please come back shortly to help with the initial launch of the new prototype."`
+				accept
+	on stopover
+		conversation
+			`Glancing around the spaceport on <stopover> you quickly spot an information desk with several busy-looking Remnant. As you approach, one of them signs a greetings to you and asks how they can be of service today. A few gestures later, and they pull out a device that they wave over the data crystal. The device makes a number of chirps before falling silent, at which the Remnant looks satisfied. "Everything looks in order, Captain <last>. Please come this way." He gestures towards an inconspicuous hallway tucked behind the information desk.`
```

Comment:
This is currently broken. It can be renamed to <stopovers>, but that would cause it to be "Glancing around the spaceport on Viminal in the Arculus system...", which would be fairly awkward when you're on the planet. Having it just be a generic "Looking around the spaceport" might be for the best. Otherwise we'd need code changes, which might not be too hard, but I'm not sure what syntax we would use for "Stopovers without the system name." <stopover planets>, I guess?

Reply:
Personally, I would love to have a <currentplanet> that completely ignores the mission specifications about destinations, sources, stopovers, etc. and just fills in *the object the pilot is currently standing on.*